doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -304,6 +304,10 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.inf.ntw.08` | Infrastructure | Network | The Architecture **must**  provide a default network which implements the Kubernetes network model. | |
 | `req.inf.ntw.09` | Infrastructure | Network | The networking solution **must not** interfere with or cause interference to any interface or network it does not own. | |
 | `req.inf.ntw.10` | Infrastructure | Network | The Architecture **must** support cluster wide coordination of IP address assignment. | |
+| `req.inf.ntw.11` | Infrastructure 	Network 	The Architecture must support multiple network interfaces to be configured for a pod. | |
```

Comment:
Not sure this is needed.  In the RM we have the following requirements covering multiple network interfaces, which is pulled through in section 2.2.1 of this chapter.

> e.cap.004 | # Connection points | number | Max number of connection points that can be assigned to a single VM or Pod by the Cloud Infrastructure

Reply:
@tomkivlin Thanks -- missed that in my search. Deleted